### PR TITLE
Tidy eval metrics, make optional.

### PR DESCRIPTION
**Pull request by mattcamp**

### CODE REVIEW

File: bundle/src/deepracer_simulation_environment/scripts/mp4_saving/training_image_editing.py [hunk 1]
```diff
@@ -58,6 +62,69 @@ def _edit_major_cv_image(self, major_cv_image, cur_training_phase):
...
+                datetime.timedelta(milliseconds=last_lap_time)) \
+                if last_lap_time != float("inf") and last_lap_time != 0 else "--:--.---"    
+            last_lap_time_text = "Last lap | {}".format(last_lap_time) 
+            major_cv_image = utils.write_text_on_image(image=major_cv_image, text=last_lap_time_text,
+                                                    loc=(width-180, loc_y+25), font=self.amazon_ember_light_18px,
+                                                    font_color=RaceCarColorToRGB.White.value,
+                                                    font_shadow_color=RaceCarColorToRGB.Black.value)
+            
+            # Progress
+            progress_text = "Progress | {}%".format(round(mp4_video_metrics_info[self.racecar_index].completion_percentage,2))
```
Comment (larsll) [inline bundle/src/deepracer_simulation_environment/scripts/mp4_saving/training_image_editing.py:100]: Cleaner is to do the rounding directly in format: 
`"Progress | {:.2f}%".format(mp4_video_metrics_info[self.racecar_index].completion_percentage)

File: bundle/src/deepracer_simulation_environment/scripts/mp4_saving/training_image_editing.py [hunk 2]
```diff
@@ -58,6 +62,69 @@ def _edit_major_cv_image(self, major_cv_image, cur_training_phase):
...
+            
+            # Progress
+            progress_text = "Progress | {}%".format(round(mp4_video_metrics_info[self.racecar_index].completion_percentage,2)) 
+            major_cv_image = utils.write_text_on_image(image=major_cv_image, text=progress_text,
+                                                    loc=(width-180, loc_y+50), font=self.amazon_ember_light_18px,
+                                                    font_color=RaceCarColorToRGB.White.value,
+                                                    font_shadow_color=RaceCarColorToRGB.Black.value)
+            
+            # Steering Angle
+            steering_text = "Steering | {}".format(mp4_video_metrics_info[self.racecar_index].steering)
```
Comment (larsll) [inline bundle/src/deepracer_simulation_environment/scripts/mp4_saving/training_image_editing.py:107]: Add rounding as above

File: bundle/src/deepracer_simulation_environment/scripts/mp4_saving/training_image_editing.py [hunk 3]
```diff
@@ -58,6 +62,69 @@ def _edit_major_cv_image(self, major_cv_image, cur_training_phase):
...
+            loc_y += 25
+            steering_text = "Throttle | {}".format(round(mp4_video_metrics_info[self.racecar_index].throttle,3))
+            major_cv_image = utils.write_text_on_image(image=major_cv_image, text=steering_text,
+                                                    loc=(loc_x, loc_y), font=self.amazon_ember_light_18px,
+                                                    font_color=RaceCarColorToRGB.White.value,
+                                                    font_shadow_color=RaceCarColorToRGB.Black.value)
+            
+            # Speed
+            loc_y += 25
+            speed_text = "Speed | {} m/s".format(utils.get_speed_formatted_str(mp4_video_metrics_info[self.racecar_index].speed))
```
Comment (larsll) [inline bundle/src/deepracer_simulation_environment/scripts/mp4_saving/training_image_editing.py:123]: Is formatting needed? Or is the input already rounded?

File: bundle/src/deepracer_simulation_environment/scripts/mp4_saving/training_image_editing.py [hunk 4]
```diff
@@ -58,6 +62,69 @@ def _edit_major_cv_image(self, major_cv_image, cur_training_phase):
...
+            # Steering Angle
+            steering_text = "Steering | {}".format(mp4_video_metrics_info[self.racecar_index].steering)
+            major_cv_image = utils.write_text_on_image(image=major_cv_image, text=steering_text,
+                                                    loc=(loc_x, loc_y), font=self.amazon_ember_light_18px,
+                                                    font_color=RaceCarColorToRGB.White.value,
+                                                    font_shadow_color=RaceCarColorToRGB.Black.value)
+            
+            # Throttle
+            loc_y += 25
+            steering_text = "Throttle | {}".format(round(mp4_video_metrics_info[self.racecar_index].throttle,3))
```
Comment (larsll) [inline bundle/src/deepracer_simulation_environment/scripts/mp4_saving/training_image_editing.py:115]: Formatting as above

**Review by larsll — CHANGES_REQUESTED** [review]

File: bundle/src/deepracer_simulation_environment/scripts/mp4_saving/training_image_editing.py (hunk 3 above)
Comment (mattcamp) [reply bundle/src/deepracer_simulation_environment/scripts/mp4_saving/training_image_editing.py]: The input is already rounded by utils.get_speed_formatted_str()

This is the way it's already displayed in existing overlays for speed.

**Review by larsll — APPROVED** [review]

**Pull request merged**